README.md

```diff
@@ -506,6 +514,17 @@ This allows to identify build errors caused by smalltalkCI updates and
 therefore helps to improve the service. It is possible to opt-out by using the
 `--no-tracking` option.
 
+### Pharo-specific Options
+
+Pharo added some specific options to customise its builds: 
+
+```yml
+env:
+  global:
+  - PHARO_VM=stable|latest              # choose stable/latest VM (default: stable)
```

Comment:
We will soon have `smalltalk_vm` support in Travis.
Instead of adding Pharo-specific env vars, can we do something like the follow?
```yaml
smalltalk_vm:
  - Pharo-7.0-stable-threaded
  - Pharo64-6.0-latest-itimer
```